src/services/models/iotHubManagerModels.js

```diff
@@ -167,3 +167,16 @@ export const toNewDeviceRequestModel = ({
         }
   };
 }
+
+export const toDevicePropertiesModel = (iotResponse, dsResponse) => {
+  const iotModel = reshape(iotResponse, {
+    'Reported': 'reported',
+    'Tags': 'tags'
+  });
+  const dsReported = getItems(dsResponse).map(({ Id }) => Id);
+  const reportedPropertiesUnion = new Set([...iotModel.reported, ...dsReported])
+  return {
```

Comment:
Taking a second look we can probably shrink this a bit more.

```
export const toDevicePropertiesModel = (iotResponse, dsResponse) => {
  const dsReported = getItems(dsResponse).map(({ Id }) => Id);
  const reportedPropertiesUnion = new Set([...iotResponse.Reported, ...dsReported]);
  return {
    tags: iotResponse.Tags,
    reported: [...reportedPropertiesUnion]
  };
}
```

Reply:
done